### DISCUSSION TITLE
R - Bagaimana cara menggunakan parameter INDEX pada fungsi findMostFreqTerms dari package tm?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#150] [Q&A] [answered]
Bagaimana cara menggunakan parameter INDEX pada fungsi findMostFreqTerms dari package tm? 
Saya baca dokumentasinya nggak ngerti-ngerti.

**Comment by NabiilahArdini** [comment, marked as answer]
Halo Bapak @Hertzspr , 

Dari dokumentasinya sepertinya menyatakan, bahwa secara default (ketika parameter `INDEX = NULL`, maka tiap dokumen (contoh: tiap sms dari set data sms) akan tentukan beberapa kata yang paling sering muncul. Namun, kita dapat mengatur agar penentuan kata yang sering munculnya dilakukan berdasarkan grup tertentu. Contoh:

```
# dicarikan frequent terms per dokumen/text
findMostFreqTerms(dtm)
```
<img width="489" alt="image" src="https://user-images.githubusercontent.com/51941102/189021057-d7855d51-47c6-4c7d-b08f-8a0787e86181.png">

```
# dicarikan frequent terms, per 10 dokumen dengan 2x pencarian (10 dokumen awal, dan 10 dokumen selanjutnya)
findMostFreqTerms(dtm, INDEX = rep(1 : 2, each = 10L))
```
<img width="473" alt="image" src="https://user-images.githubusercontent.com/51941102/189021316-9e5a7ee8-6ac1-4dda-a697-f921e163fb3b.png">

Demikian yang saya coba pahami. 
Semoga dapat membantu!

**Reply by Hertzspr** [reply]
Terima kasih.
